docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+_Openstack Users_.
+
+![Add a new OpenStack user](assets/ostack-cli/shot-01.png)
+
+Then, at the top right-hand side of the {{extra.gui}}, click once more 
+the _Add new Openstack user_ option. A new pane will slide into view, 
+titled _Create Openstack User_.
+
+![Create new OpenStack user](assets/ostack-cli/shot-02.png)
+
+Type in a username and a password for the new OpenStack user. To ensure 
+you typed the password correctly, you must re-type it below. This 
+password should be adequately strong, and thus a password manager may 
+come in handy.
+
+![Type-in username and password](assets/ostack-cli/shot-03.png)
+
+Scroll down a bit, so the _Regions_ section is in full view. Expand one 
+or more of the available regions you want your new user to have access 
+to. For each one of the expanded regions, select one or more 
+_Projects_. For each project, activate one or more _Roles_. (Hint: For 
+an overview of the rights different available roles provide, hover the 
+mouse pointer over the exclamation mark icon by the _Roles_.) 
+
+![Regions, projects, and roles](assets/ostack-cli/shot-04.png)
+
+Optionally, type in a description for the new OpenStack user. Then, 
+create the user by clicking the green _Create_ button below the 
+_Description_ box.
+
+![User description and creation](assets/ostack-cli/shot-05.png)
+
+The new OpenStack user will be ready in just a few seconds. At any 
+time, you can view all available OpenStack users by going to the 
+left-hand side pane on the {{extra.gui}} and selecting _Users_ > 
+_Openstack Users_.
+
+![All available OpenStack users](assets/ostack-cli/shot-06.png)
+
+## Creating and downloading an RC file
+
+On the {{extra.gui}} expand the left-hand side pane, click _Users_ and 
+then _Openstack Users_. You will then see, listed in the main pane 
+titled _OpenStack Users_, all available users. Click on the 
+three-dotted round icon on the right of the user you want to create an 
+RC file for. From the pop-up that appears, select _Generate RC file_.
+
+![Generate  RC file](assets/ostack-cli/shot-07.png)
+
+The RC file will be generated automatically. Before downloading it onto 
+your local computer, you must select one of the available projects to 
+relate the RC to. Do so and then click the blue _Download_ button.
+
+![Select project and download](assets/ostack-cli/shot-08.png)
+
+A "Save as" dialog window appears, specific to the Operating System you 
+are currently using. Select a convenient location and save your RC file.
+ 
+## Modifying and sourcing the RC file 
+
+The general naming for RC files goes like this:
+
+```plain
+your_username--region_name--project_name--rc
+```
+
+So, assuming your username is `olafsdottir`, and the RC file has been 
+created for the `fra1` region and the `katla` project, your RC file 
+name should be this:
+
+```plain
+olafsdottir--fra1--katla--rc
+```
+
+Take a look at the contents of this file --- they should be like this:
+
+```plain
```

Comment:
This should use `bash`, not `plain` syntax highlighting.

Reply:
You know, at first I thought to use `bash`, but then I thought we are showing the contents of a text file and opted for `plain`. Anyhoo, let's go with your suggestion (and my initial thought).